src/List.mo

```diff
@@ -39,6 +81,15 @@ module {
 
   /// Treat the list as a stack.
   /// This function combines the `head` and (non-failing) `tail` operations into one operation.
```

Comment:
Remove the head of `list`, returning the element and the tail of the list in a pair.